pkg/microservice/aslan/core/environment/service/environment.go

```diff
@@ -171,41 +171,55 @@ func GetProductStatus(productName string, log *zap.SugaredLogger) ([]*EnvStatus,
 	return envStatusSlice, err
 }
 
-func ListProducts(productNameParam string, userName string, log *zap.SugaredLogger) (resp []*ProductResp, err error) {
-	products, err := commonrepo.NewProductColl().List(&commonrepo.ProductListOptions{Name: productNameParam, IsSortByProductName: true})
+func ListProducts(projectName string, envNames []string, log *zap.SugaredLogger) ([]*ProductResp, error) {
```

Comment:
this is a common db semantic，it is not appropriate to put it in the service level , i think it will pollute  the service code with db code